src/ring_theory/algebra_tower.lean

```diff
@@ -28,17 +28,16 @@ universes u v w u₁
 variables (R : Type u) (S : Type v) (A : Type w) (B : Type u₁)
 
 /-- Typeclass for a tower of three algebras. -/
-class is_algebra_tower [comm_semiring R] [comm_semiring S] [semiring A]
-  [algebra R S] [algebra S A] [algebra R A] : Prop :=
-(smul_assoc : ∀ (x : R) (y : S) (z : A), (x • y) • z = x • (y • z))
+abbreviation is_algebra_tower [comm_semiring R] [comm_semiring S] [semiring A]
```

Comment:
Do you really want to keep this abbreviation? It seems simpler to just use `is_scalar_tower` everywhere.

Reply:
I completely agree with this. Working with both names for derivations I already see that having to remember which name is used where is confusing and I do not see the point of having this abbreviation.

Reply:
Opened #3785.